fdbserver/workloads/MachineAttrition.actor.cpp

```diff
@@ -409,7 +409,9 @@ struct MachineAttritionWorkload : FailureInjectionWorkload {
 						CODE_PROBE(true, "Marked a zone for maintenance before killing it");
 						wait(success(
 						    setHealthyZone(cx, targetMachine.zoneId().get(), deterministicRandom()->random01() * 20)));
-					} else if (BUGGIFY_WITH_PROB(0.005)) {
+					} else if (!g_simulator->willRestart && BUGGIFY_WITH_PROB(0.005)) {
```

Comment:
If this applies only to the first test being run, then we still may be subjected to this for any old binary that doesn't have this fix, is that right? Do we know what versions are impacted by this?

Is there any way we can change the parameters of the test for older versions that would avoid this problem?